sphinx-docs/usermanual/userman_admin.rst

```diff
@@ -204,30 +218,101 @@ Downloading Individual Videos
 #. Click the "Get Language Pack" button. 
-#. Once the download finishes, you can see your language pack in the list of installed packs. Students and coaches will now be able to switch their language to any of the installed language packs. Their default will be the default that you set by clicking on "Set as default". 
+#. Once the download finishes, you can see your language pack in the list of installed packs. Learners and coaches will now be able to switch their language to any of the installed language packs. Their default will be the default that you set by clicking on "Set as default". 
 
 Delete Language Packs
 ^^^^^^^^^^^^^^^^^^^^^^^^^
+To delete language packs:
+
 #. Log in as the administrator.
 #. Click the "Languages" link in the navigation bar
 #. In the Installed Languages section, there is a button for deletion of each language.
 
 
+Restarting Your Server
+-----------------------
+If you have made some configuration changes (such as changing the filepath to your video content to your liking), or if you feel the need to reboot your KA Lite system, you may want to restart your server. Please note that this will cause KA Lite to become inaccessible to any users. However, this will not delete any user accounts or information that you have configured during set up. 
+
+This process varies, depending on which OS you are running the KA Lite Server on. 
+
+Restarting Your Server: Windows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+Restarting Your Server: Linux
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+Restarting Your Server: Mac 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#. Open up your terminal. You may do this by navigating to the magnifying glass |magglass| at the top right corner of your screen, and typing in "Terminal", then hitting "Enter" on your keyboard.
+
+    .. image:: search.png
+        :align: center
+        :width: 700
+        
+#. Your terminal should be opened up. It should look a little something like the following: 
+
+    .. image:: terminal.png 
+        :align: center
+
+    Type in "sudo lsof -i :8008" into the prompt, without the quotes. Running this command will prompt you for the machine password. Please enter your machine password, then hit "ENTER" on your keyboard.
+
+#. After entering your password, some information should appear on your screen. Copy the number that appears under "PID". Please note that the number shown will vary, and may not be exactly the same as the one shown in the picture. Loosely speaking, this is the process ID that is currently running your server. We will need its ID in order to stop the process -- in turn, stopping the KA Lite server. 
+
+    .. image:: process.png 
+        :align: center
+
+#. Type in "sudo kill -9 <enter PID here>", without quotes into the terminal. Type in the PID that you just copied from the previous step. Be sure to do this without the "< >" symbols. For example, if the PID was the same as the one in the above photo example, you would type in "sudo kill -9 9585". 
```

Comment:
Instead of qualifying code statements to be without quotes, it might be a good idea to format them differently. For example, double backticks as suggested here: http://sphinx-doc.org/rest.html#inlinemarkup

Then you can type code examples literally because they're distinguished from surrounding text.